# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchStreamUtils.h
# Commit: b9d73f961292
# Full Hash: b9d73f9612929dda88ca7e68d8d2e04de2aa2a86
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-20 21:21:03
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 989cbb08b87e -r b9d73f961292 dom/fetch/FetchStreamUtils.h
--- a/dom/fetch/FetchStreamUtils.h	Fri Jan 20 09:09:17 2023 +0000
+++ b/dom/fetch/FetchStreamUtils.h	Fri Jan 20 09:09:17 2023 +0000
@@ -40,7 +40,8 @@
 // process. Can only be called in the parent process.
 ParentToChildStream ToParentToChildStream(
     const NotNull<nsCOMPtr<nsIInputStream>>& aStream, int64_t aStreamSize,
-    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent);
+    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent,
+    bool aSerializeAsLazy = true);
 
 // Convert a ParentToParentStream to a ParentToChildStream. Can only be called
 // in the parent process.